app/src/main/java/org/openmrs/eip/app/management/entity/DebeziumEvent.java

```diff
@@ -0,0 +1,50 @@
+package org.openmrs.eip.app.management.entity;
+
+import javax.persistence.AttributeOverride;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import org.openmrs.eip.component.entity.Event;
+
+@Entity
+@Table(name = "debezium_event_queue")
+public class DebeziumEvent extends AbstractEntity{
+	private static final long serialVersionUID = -1884382844867650350L;
+	
+    @Embedded
+    @AttributeOverride(name = "identifier", column = @Column(updatable = false))
+    @AttributeOverride(name = "primaryKeyId", column = @Column(name = "primary_key_id", nullable = false, updatable = false))
+    @AttributeOverride(name = "tableName", column = @Column(name = "table_name", nullable = false, updatable = false, length = 100))
+    @AttributeOverride(name = "operation", column = @Column(nullable = false, updatable = false, length = 1))
+    @AttributeOverride(name = "snapshot", column = @Column(nullable = false, updatable = false))
+    private Event event;
```

Comment:
Why are you embedding this Event class? We don't always have the uuid, this true when processing subclass tables like patient

Reply:
Just because I don't want to rewrite the Event as DebeziumEvent. In the end, these two classes are very same...

Reply:
Is it because you need to capture the uuid?